packages/fx-core/resource/package.nls.json

```diff
@@ -933,7 +939,8 @@
   "plugins.bot.SomethingNotFound": "%s is not found.",
   "plugins.bot.SomethingNotExisting": "%s is not existing.",
   "plugins.bot.FailedToCreate": "Failed to create %s",
-  "plugins.bot.FailedToGetAlreadyCreatedBot": "Could not get bot with id %s which was created before. \nThis could happen if the bot was created by another account. Visit %s to learn more.",
+  "plugins.bot.FailedToGetAlreadyCreatedBot": "Could not create bot with id %s which may be taken by an existing bot created by another account before. Visit %s to learn more.\nOr API call to Developer Portal failed: Error, Request failed with status code 500, API name: create-bot, X-Correlation-ID: undefined. This may come from some intermittent service error. Please wait for a few minutes and retry the current step.",
```

Comment:
Could not > Cannot 
don't need this "created by another account before", consider remove it.
suggest to remove "the current step" to make message shorter.

Reply:
Good suggestions to make better wording. Will do.